ci/test/metabase-demo.compose.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Map from host-port:internal port
+#
+# This mostly just shows all the ports that are available to the host system, if you want
+# to change these you must restart the docker-compose cluster.
+x-port-mappings:
+  - &metabase 3000:3000
```

Comment:
I would just inline this bit. We don't need to support configurability in the CI-specific docker compose files, since they're not intended to be run by users (outside of debugging CI failures).